biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -533,13 +580,26 @@ real(r8) :: tcwood             ! Temperature response function for wood
                      ! the sapwood pools.
                      ! Units are in (kgN/plant)
                      ! ------------------------------------------------------------------
-                     live_stem_n = EDPftvarcon_inst%allom_agb_frac(currentCohort%pft) * currentCohort%bsw / &
-                           woodcn(currentCohort%pft)
-                     live_croot_n = (1.0_r8-EDPftvarcon_inst%allom_agb_frac(currentCohort%pft)) * currentCohort%bsw  / &
-                           woodcn(currentCohort%pft)
-                     froot_n       = currentCohort%br / frootcn(currentCohort%pft) 
-                     
+
+                     select case(hlm_parteh_model)
```

Comment:
add something like "read in tissue N contents from PARTEH model"

Reply:
ok, will do